hardhat.config.ts

```diff
@@ -17,8 +17,30 @@ if (walletSecret === 'undefined') {
 }
 const accounts = walletSecret.length === 64 ? [walletSecret] : { mnemonic: walletSecret };
 
-const mainnetEtherscanKey = process.env.MAINNET_ETHERSCAN_KEY;
-const testnetEtherscanKey = process.env.TESTNET_ETHERSCAN_KEY;
+let testnetEtherscanKey;
```

Comment:
I'd avoid using the ethscan in the name (maybe `scan` will do) since we're using gnosis scan for the mainnet.

`let testnetEtherscanKey;` -> `let testnetEtherscanKey: number;`
`let mainnetEtherscanKey;` -> `let mainnetEtherscanKey: number;`